From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Sunu Aziz Rahayu <sunuazizrahayu@users.noreply.github.com>
Date: Fri, 26 Sep 2026 00:00:00 +0700
Subject: [PATCH] lineage: disable Lineage charger animation for GSI

vendor/lineage/config/common_mobile.mk selalu menarik
lineage_charger_animation yang menginstal battery_fail.png ke
product/etc/res/images/charger, bertabrakan dengan modul charger
milik device/phh/treble sehingga kati gagal:

  overriding commands for target
  `.../system/product/etc/res/images/charger/battery_fail.png'

Charger bawaan treble sudah cukup untuk GSI.

---
 lineage.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/lineage.mk b/lineage.mk
index 0000000..0000000 100644
--- a/lineage.mk
+++ b/lineage.mk
@@ -13,5 +13,8 @@ TARGET_BOOT_ANIMATION_RES := 1080
 # APN
 PRODUCT_PACKAGES += apns-conf.xml

+# Treble already provides its own charger images
+WITH_LINEAGE_CHARGER := false
+
 #Oplus fingerprint
 $(call soong_config_set,surfaceflinger,udfps_lib,//hardware/oplus:libudfps_extension.oplus)
--
2.54.0
